src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -59,10 +63,22 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     }
 
     List<Message> messages = datastore.getMessages(user);
+
+    String targetLanguageCode = request.getParameter("language");
+    if (targetLanguageCode != null) {
+      translateMessages(messages, targetLanguageCode);
+    }
+
     Gson gson = new Gson();
     String json = gson.toJson(messages);
 
     response.getWriter().println(json);
+
+    // Gets target language & calls helper function to translate
+    String targetLanguageCode = request.getParameter("language");
```

Comment:
How does this work? Isn't this defined above already?

Reply:
Realized I included it twice. Removed in next commit